pkg/session_recording/recordings.jsx

```diff
@@ -340,7 +349,7 @@
          * Set the cockpit location to point to the specified recording.
          */
         navigateToRecording(recording) {
-            cockpit.location.go([recording.id]);
+            cockpit.location.go([], $.extend(cockpit.location.options, { id: recording.id }));
```

Comment:
I think it should be something like http://localhost:9090/session_recording?dateSince=20170129&user=user1#/0777dde78ccb4265862ab18d1857bbfd-503-39d26 but yes.